.env

```
@@ -0,0 +1,16 @@
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)